pkg/util/template/template.go

```diff
@@ -11,7 +12,11 @@ var templateValidator = NewValidator(
 	MaxDepth(15),
 )
 
-var templateFuncMap = map[string]interface{}{
-	messageformat.TemplateRuntimeFuncName: messageformat.TemplateRuntimeFunc,
-	"makemap":                             MakeMap,
+func MakeTemplateFuncMap() map[string]interface{} {
+	var templateFuncMap = sprig.FuncMap()
```

Comment:
Use `HermeticHtmlFuncMap`/`HermeticTextFuncMap` to avoid security problem. (`FuncMap` has `env`, `getHostByName`, etc.)

Reply:
Fixed